README.md

```diff
@@ -87,3 +87,10 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+
+ <hr>
+
+<a href="https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782" target="_blank"> Figma Designs </a>
+
+</br>
+
```

Comment:
export the Home page and attached it here

Reply:
Please give me access to the figma designs

Reply:
now you can access the figma file

Reply:
@acenasir

Reply:
Added Home page png export into root folder

Reply:
@acenasir Please delete the Home.png and update README file using GitHub file update. You can drag the home image and drop it in to the bottom of the README file and wait until it update, then send a pull request.

Reply:
@acenasir Please delete the Home.png and update README file using GitHub file update. You can drag the home image and drop it in to the bottom of the README file and wait until it update, then send a pull request.